dnf5/CMakeLists.txt

```
@@ -2,6 +2,8 @@ if(NOT WITH_DNF5)
     return()
 endif()
 
+set(SYSTEMD_SYSTEM_UNIT_DIR /usr/lib/systemd/system)
```

We should get this from pkgconfig by requesting the `systemdsystemunitdir` variable from `systemd` pc file.

```suggestion
pkg_check_modules(SYSTEMD REQUIRED systemd)
pkg_get_variable(SYSTEMD_SYSTEM_UNIT_DIR systemd systemdsystemunitdir)
```

---

Thanks!